flow/layers/image_filter_layer.cc

```diff
@@ -64,6 +64,7 @@ void ImageFilterLayer::Preroll(PrerollContext* context,
     // from frame to frame so we try to cache the layer itself
     // for maximum performance.
     TryToPrepareRasterCache(context, this, matrix);
+    context->save_layer_count += 1;
```

Comment:
What saveLayer is this counting for?

Reply:
Rasterizing this layer for the cache is effectively a saveLayer - although your point about this possibly touching the cache without updating the image is well taken.

Reply:
I can imagine that the hit rate on this specific count being true is probably very low. But it can vary case by case.

Reply:
Also, this increment only accounts for the saveLayer in this layer's own Paint() call. Consider that this layer has children and those children can also contain saveLayer calls and this increment of 1 can't even hope to account for all of those.

While we could design another accounting workaround for that issue, this is perhaps yet another reason that using trace events localized at the sites of the saveLayer calls would greatly improve the accuracy of these metrics in a less convoluted way.